tool/dartformat.sh

```diff
@@ -23,6 +23,29 @@ if [[ $NULL_SAFETY == 1 ]]; then
   dart format $* `find $NULL_SAFETY_EXAMPLES -name "*.dart" \
       ! -path "**/.*" \
       ! -path "**/build/**"`
+
+  dart format -l 60 \
```

Comment:
What's the reason for `-l 60` here?

Reply:
We decided a long time ago that these examples needed a shorter line length, probably to avoid scrolling with common page widths.